docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FastForest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class FastForest
+    {
+        // This example requires installation of additional NuGet package
+        // <a href="https://www.nuget.org/packages/Microsoft.ML.FastTree/">Microsoft.ML.FastTree</a>.
+        public static void Example()
+        {
```

Comment:
Is it possible to execute those `Example()`s in a test? Just want to make sure they are always executable.

Reply:
Good idea. I'll add tests in a separate PR.

---
In reply to: [267441145](https://github.com/dotnet/machinelearning/pull/3035#discussion_r267441145) [](ancestors = 267441145)